tests/dragonfly/replication_test.py

```diff
@@ -74,7 +74,6 @@ async def test_replication_all(
         args["maxmemory"] = str(t_master * 256) + "mb"
 
     if big_value:
-        args["compression_mode"] = 0
```

Comment:
@chakaz now we can enable `serialization of big values` by default. Any ideas what a good value is ?

Reply:
I would say 10mb, but @romange this is your call

Reply:
yeah, 10MB looks good 👍🏼

Reply:
I will turn this on with the memory test I will push soon